c++/src/linked-list.cc

```diff
@@ -254,6 +263,11 @@ int main() {
   list.printTo(listString);
   assert(listString.str() == "0 → 1 → 2 → 3 → 4 → 5 → 6"); listString.str("");
 
+  LinkedList<int> initializer_list_list = LinkedList<int>{0, 1, 2, 3};
+  initializer_list_list.printTo(listString);
+  assert(listString.str() == "0 → 1 → 2 → 3");
+  listString.str("");
+
```

Comment:
This would've been a good opportunity to make this an isolated test in the manner I have described extensively. Even if you don't go and isolate every test, at least doing it with this and future tests would've been great.